lib/recurly/client.rb

```diff
@@ -53,16 +53,8 @@ def initialize(api_key:, site_id: nil, subdomain: nil, **options)
       yield(self) if block_given?
     end
 
-    def next_page(url)
-      path = extract_path(url)
-      request = Net::HTTP::Get.new path
-      set_headers(request)
-      http_response = run_request(request)
-      handle_response! request, http_response
-    end
-
-    def get_resource_count(url)
-      request = Net::HTTP::Head.new url
+    def get_resource_count(path, options)
```

Comment:
Can't we also get rid of this? and move it into the `Pager`? It could call a `head` method in the client.